user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -4,91 +4,205 @@ uid: DashboardGrid
 
 # Grid
 
-> [!IMPORTANT]
-> The grid component is in preview until DataMiner 10.4.1/10.5.0. If you use the preview version of the feature, its functionality may be different from what is described below. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+Available from DataMiner 10.4.1/10.5.0 onwards<!--RN 34761-->.
 
-Available from DataMiner 10.4.1/10.5.0 onwards<!--RN 34761-->. Prior to this, the component is available in soft launch from DataMiner 10.2.12 onwards, if the soft-launch option *ReportsAndDashboardsDynamicVisuals* is enabled.
+The grid component is a versatile visualization designed to display the results of queries in a clear, tile-based grid format. It offers a **wide range of customization options**, including dynamic scaling, conditional formatting, and interactive features that let you trigger actions directly from each grid block.
 
-This component allows you to visualize data as a grid.
+![Grid](~/user-guide/images/Grid.png)<br>*Grid component in DataMiner 10.5.6*
 
-![Grid](~/user-guide/images/Grid.png)<br>*Grid component in DataMiner 10.4.1*
+With this component, you can:
```

Comment:
One of the main capabilities of the grid is not listed here. Which is to act as a selector. Even more than a table, a grid is used to allow a selection. This selection is then used to steer other functionality.